src/theme.ts

```diff
@@ -222,6 +222,8 @@ export const theme360 = extendTheme({
           700: '#1C75B3',
           800: '#1365A2',
           900: '#014883',
+          A100: '#013865',
```

Comment:
What are these two colors for?

Reply:
I removed both A100 and A200 because A100 was being used in confirmationRows which isn't being used anymore and A200  was a color that can be replaced with primary blue